PointForRRT input_point
---
PointArray path_to_follow
